website/src/components/BlogCard/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import useStyles from "./style";
+import { Button, Card, CardActions, CardContent, CardMedia } from "@material-ui/core";
+import BlogImage from "../BlogImage";
+import { useHistory } from "react-router-dom";
+import DisplayAuthorandReadTime from "../DisplayAuthorandReadTime";
+import ReactMarkdown from "react-markdown";
+import { getContentPreview } from "../../utils/getContent";
+import CustomTag from "../CustomTag";
+
+interface BlogCardProps {
+    isAuthorPage: boolean;
+    blog: { title: string; author: string; excerpt: string, author_info: string; date: string; tags: Array<string>; content: string; id: number; slug: string; }
+    handleTagSelect: any;
```

Comment:
Can we provide a type here like this ; 
https://www.carlrippon.com/Different-ways-to-strongly-type-function-component-props-with-typescript/

Reply:
Done